Answers/hw03_conwellddw.py

```diff
@@ -0,0 +1,121 @@
+#################################################################################
+# Author: Dayton Conwell
+# Username:conwelld
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:to draw a unique object
+# Google Doc Link:https://docs.google.com/document/d/16yg3ch82DAR6bArUB5gwlDlm5b43f2yEVE2Y9IcKr5g/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# Professor Scott Hegen
+#
+#################################################################################
+
+import turtle
+
+wn = turtle.Screen() #initialize screen window
+wn.bgcolor('grey') #sets bg color to gray
+turtle.colormode(255) #allows for rgb values to be input for color
+
+def foundation(base):
+    base.goto(0,-200)
+    base.fillcolor('black')
+    base.begin_fill()
+    for i in range(4): #makes the base of the house and fills it black
+        base.forward(100)
+        base.left(90)
+        base.forward(100)
+    base.end_fill() #ends the filling
+
+def roof(base):
+    base.penup()
+    base.goto((-100.00,0.00)) #draws the shape of the roof (triangle) and fills it black
+    base.pendown()
+    base.left(45)
+    base.fillcolor('black')
+    base.begin_fill()
+    base.forward(142)
+    base.right(90)
+    base.forward(142)
+    base.end_fill()
+def window(base):
```

Comment:
Pay attention to the gray squigglies, which are style guidelines. They indicate there should be a blank line before any function definition (for readability).